recipes/openspecfun/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "openspecfun" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "1505c7a45f9f39ffe18be36f7a985cb427873948281dbcd376a11c2cd15e41e7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/openspecfun/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc             # [osx]
+    - libgfortran     # [linux]
+    - openlibm 0.5.4
+
+  run:
+    - libgfortran
+    - openlibm 0.5.4
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/Faddeeva.h"                             # [unix]
```

Comment:
I'm not sure whether this is the best place to have this as we might package `Faddeeva` separately.

Reply:
If we do, we can always make this depend on a `faddeeva` package.